server/templates/static/homepage.html

```diff
@@ -74,7 +74,7 @@ <h4>Data from</h4>
       </article>
       <article>
         <p>
-          Over 240B data points across 450k variables from the
+          Over 240B data points across 260k variables from the
```

Comment:
nit: (can feel free to ignore or slide into another PR) `k` -> `K` for symmetry with `B` for billion?

Reply:
+1 :-)

Reply:
Great call out! Put up https://github.com/datacommonsorg/website/pull/4308